blocks/columns/columns.js

```diff
@@ -13,6 +15,12 @@ export default function decorate(block) {
           picWrapper.classList.add('columns-img-col');
         }
       }
+      if (block.classList.contains('core-team') && !col.classList.contains('columns-img-col')) {
```

Comment:
Statement "block.classList.contains('core-team')" is executed in loop, suggest to put it in an object outside loop and use object.

Reply:
@mkbansal1 I will remove this JS. This is not required.